website/docs/developer/research/research-lambda-container.md

```diff
@@ -0,0 +1,139 @@
+---
+id: research-lambda-container
+title: Using Lambda functions as container research Notes
+description: Research Notes on 
+---
+
+## Overview
+
+Lambda functions can now be deployed as container images using Docker instead of zip files. This research webpage discusses how lambda functions can be prototyped as container and its pros and cons.
+
+## Pros and Cons of using container for lambdas
+
+### Pros
+- Defining runtime environment in a container image gives developers more control over their environment compared to what they get with predefined runtimes and zipping dependencies. 
+- you can now package and deploy Lambda functions as container images of up to 10 GB in size compared to only 250MB in case of lambda deployment package size. 
+- preferable for data-heavy or dependency-heavy applications.
+- The Lambda service provides a variety of base image options with pre-installed runtimes that will be patched and maintained by AWS.
+- There is no additional cost to use the container feature but you have to pay for the ECR repository.
```

Comment:
The con mentioned at the end feels awkward. Try reworking to something like "Once deployed, containerized lambdas have no additional cost compared to zipped lambdas."

Reply:
modified wording.

Reply:
I still think you could drop the mention of the con, and even the entire point if you want. You cover costs in the Cons section.

Reply:
removed `cost` part from Cons section

Reply:
But the cost is a con...

Reply:
the cost is both a pros and cons

`pros` because it requires no additional cost to use container

`cons` because it requires ECR which has addtional cost.

Reply:
I guess I don't see "no change" as an argument in favor of using containers. Especially as the only way we know to get this working does incur an additional cost to use containers.